corehq/apps/cloudcare/templates/form_entry/templates.html

```diff
@@ -222,7 +222,8 @@ <h3 class="caption" data-bind="html: header" tabindex="0"></h3>
       </div>
     </div>
     <div class="panel-footer">
-      <button class="btn btn-default btn-xs add" href="#" data-bind="click: newRepeat">{% trans "Add new repeat" %}</button>
+      <button class="btn btn-default btn-xs add" href="#" data-bind="click: newRepeat, template: { afterRender: afterRender }"
```

Comment:
I'd get rid of `{% trans "Add new repeat" %}` below, and get rid of template/afterRender, and instead use a simple

```
text: getTranslation('repeat.dialog.add.new')
```

and then define `self.getTranslation` above instead of defining `self.afterRender`

Reply:
Done.